### PR TITLE
update Get points and rework styles for images

### PR DESCRIPTION
**Pull request by nrosto**
https://github.com/users/kaelzeer/projects/1?pane=issue&itemId=57353671
- added styles for vertical and horizontal images
- update get points for /api to /api/posts
- update img src
- deleted <a> in postBlocks
- added if for post.images

### CODE REVIEW

File: frontend/.gitignore [hunk 1]
```diff
@@ -34,3 +34,6 @@ yarn-error.log*
 # typescript
 *.tsbuildinfo
 next-env.d.ts
+
+# testImage
+/public
```
Comment (kaelzeer) [inline frontend/.gitignore]: @nrosto 
уже не нужно же?

**Review by kaelzeer — CHANGES_REQUESTED** [review]
есть фидбек

**Review by kaelzeer — APPROVED** [review]
@nrosto  аппрув

**Comment by nrosto** [comment]
тема

**Pull request merged**